src/test/java/pages/constructor/ConstructorPage.java

```diff
@@ -38,21 +40,29 @@ public SelenideElement order() {
     public void makeOrder() {
         order().click();
         try {
-            sleep(25000);
+            sleep(18000);
         } catch (Exception e) {
             e.printStackTrace();
         }
+        order().pressEscape();
     }
-public SelenideElement orderNumber() {
-        return element(ORDER_NUMBER);
-    }
-    public void closeOrderNumber(){
-        orderNumber().click();
-    }
+
     public SelenideElement orderFeed() {
         return element(ORDER_FEED);
     }
+
     public void clickOrderFeed() {
         orderFeed().click();
```

Comment:
orderFeed().shouldBe(visible.because("")).click();
Должно быть так, возможно это прям так не заработает, придумай способ это реализовать без try/catch